README.md

```diff
@@ -134,3 +123,64 @@ No modules.
 You can report issues and request features for this module in GitHub issues in the module repo. See [Report an issue or request a feature](https://github.com/terraform-ibm-modules/.github/blob/main/.github/SUPPORT.md).
 
 To set up your local development environment, see [Local development setup](https://terraform-ibm-modules.github.io/documentation/#/local-dev-setup) in the project documentation.
+
+<!-- BEGIN_TF_DOCS -->
```

Comment:
please remove this section - we have our own metadata tags (which you incorrectly removed) for terradocs